src/Gui/NaviCube.cpp

```diff
@@ -767,40 +815,40 @@ void NaviCubeImplementation::initNaviCube(QtGLWidget* gl) {
 	z = r45x * z;
 	x = r45x * x;
 	z *= 1.25f; // edge face position
```

Comment:
Calculate using gap instead:
```suggestion
	z *= sqrt(2) * (1 - gap);
```

Reply:
Thanks, this is now in: commit f70329390cea